File: C_Hero_Academy.cpp
#include<bits/stdc++.h>
using namespace std;
#define ll long long

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    ll t;
    cin>>t;
    while(t--) {
        ll n;
        cin>>n;
        vector<ll> arr(n);
        for(ll i=0;i<n;i++) {
            cin>>arr[i];
        }
        sort(arr.begin(),arr.end());
        ll avg=arr[n/2];
        ll ans=0;
        for(ll i=0;i<n;i++) {
            ans+=llabs(avg-arr[i]);
        }
        cout<<ans<<endl;
    }
}   

File: C_Hero_Academy.py
t=int(input())
while t!=0 :
    t=t-1
    n=int(input())
    arr = list(map(int, input().split()))
    sum=0
    for i in arr:
        sum+=i

    avg = sum/n
    ans=int(0)
    for i in arr:
        ans+=abs(avg-i)
    print(int(ans))

    
    

    